README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
+
+Past sporks are loaded from the filesystem using Flow's Write-Ahead Log to replay the changes to all registries, and
+updates are pushed directly to the DPS by the execution nodes in order to handle live networks.
+
+The Flow DPS maintains different specialized indexes for different purposes. One index is used for accessing the entire
```

Comment:
`multiple specialized indexes`

Reply:
```suggestion
The Flow DPS maintains multiple specialized indexes for different purposes. One index is used for accessing the entire
```

Reply:
Maybe we can even get rid of the `multiple` here, as it's implied by the plural form. WDYT?

Reply:
Yeah, I don't think it adds clarity, but it doesn't hurt to keep it either.